src/VisualStudio/Core/Def/EditorConfigSettings/NamingStyle/ViewModel/ColumnViewModels/NotifyPropertyChangedBase.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace Microsoft.VisualStudio.LanguageServices.EditorConfigSettings.NamingStyle.ViewModel
+{
+    internal abstract class NotifyPropertyChangedBase : INotifyPropertyChanged
```

Comment:
We really need just a shared implementation of NPC. Maybe this could finaly be it???

Reply:
https://github.com/dotnet/roslyn/issues/58148